### ISSUE TITLE
[Feature] Bypass iOS add to Home Screen prompt

### ISSUE THREAD

**Issue by ILikePlayingGames** [redphx/better-xcloud#52]
On iOS, xCloud requires that it be added to the Home Screen as a portable web app before being able to play, thereby skipping better xcloud. Is it possible to add a setting to bypass this?

**Comment by redphx** [comment]
I think switching User-Agent to "Edge on Windows" works

**Comment by ILikePlayingGames** [comment]
Just tested, it did not:(. Also I met the 404 bug again :(

**Comment by ILikePlayingGames** [comment]
I think it might use `window.navigator.platform` for OS detection the last time I checked

**Comment by redphx** [comment]
Let me check when I have time

**Comment by ILikePlayingGames** [comment]
Thanks!

**Comment by redphx** [comment]
@ILikePlayingGames  
Try adding these lines at the end of the userscript with the default User-Agent (switch to Edge if it doesn't work)  

```js
Object.defineProperty(window.navigator, 'standalone', {
    value: true,  // change to "false" if it doesn't work
});
```  

If it still doesn't work then try to add these lines:  
```js
window.addEventListener('beforeinstallprompt', function(e) {
    e.preventDefault();
    return false;
});
```

I have an iPad but it's running iPadOS 14 so I couldn't install the Userscripts extension.

**Comment by F1gmentS16** [comment]
Is it possible to remove the url-line at the top?

**Comment by ILikePlayingGames** [comment]
@redphx the first option worked. Thanks!

**Comment by redphx** [comment]
@ILikePlayingGames set to `true` with default UA?  

@F1gmentS16 no unless you turn on fullscreen mode.

**Comment by redphx** [comment]
@F1gmentS16 you can try this  

![image](https://github.com/redphx/better-xcloud/assets/96280/c83832cd-dacc-4762-b584-9ea019d8800e)

**Comment by redphx** [comment]
Implemented in [version 1.9](https://github.com/redphx/better-xcloud/releases/tag/v1.9)

**Comment by ILikePlayingGames** [comment]
> @ILikePlayingGames set to `true` with default UA? 

Yes
